engine/Assets/Scripts/UI/Dynamic/Panels/Information/ScoreboardPanel.cs

```diff
@@ -18,8 +19,8 @@ public class ScoreboardPanel : PanelDynamic {
 
     private readonly bool _showTimer;
 
-    private Label time, redScore, blueScore;
-    private Content topContent, bottomContent;
+    private static Label _time, _redScore, _blueScore, _redLabel, _blueLabel;
```

Comment:
Should these be static? If AssignColors is made not static, then I don't think these would have to be

Reply:
Maybe AssignColors could be made not static, but I don't think I'm knowledgeable enough to confidently make that change, so for now, the variables need to be static so there won't be errors.